website/docs/r/workspace.html.markdown

```diff
@@ -68,6 +68,10 @@ The following arguments are supported:
 * `assessments_enabled` - (Optional) Whether to regularly run health assessments such as drift detection on the workspace. Defaults to `false`.
 * `auto_apply` - (Optional) Whether to automatically apply changes when a Terraform plan is successful. Defaults to `false`.
 * `auto_apply_run_trigger` - (Optional) Whether to automatically apply changes for runs that were created by run triggers from another workspace. Defaults to `false`.
+* `auto_destroy_at` - (Optional) A future date/time string at which point all resources in a workspace will be scheduled for deletion. Must be a string in RFC3339 format (e.g. "2100-01-01T00:00:00Z").
```

Comment:
I'm trying to imagine how a static date could be useful in the context of config. Likely this data would come from a variable, but aside from that it will be inevitable that they will define a relative time that will end up changing with every plan.

`timeadd(timestamp(), "1h")`